src/Security/Voter/RealmSigningLogVoter.php

```diff
@@ -36,10 +40,24 @@ protected function voteOnAttribute(string $attribute, mixed $subject, TokenInter
             return false;
         }
 
-        assert($subject instanceof RealmSigningLog);
+        if (in_array('ROLE_SUPER_ADMIN', $user->getRoles())) {
+            return true;
+        }
+
+        assert(
+            $subject instanceof RealmSigningLog ||
+            $subject instanceof Realm,
+        );
```

Comment:
In een eerdere MR al benoemd dat het is mss netter is om eigen Voter per Entity op te tuigen.

Reply:
In een later PR is er zelfs nog een class bijgekomen. Zal het oppakken.